pmm-tests/pmm-2-0-bats-tests/helm-test.bats

```diff
@@ -0,0 +1,111 @@
+## pmm k8s helm tests
+### needs: helm, kubectl, k8s cluster, default kubeconfig
+
+setup() {
+
+    PROJECT_ROOT=$(dirname "$BATS_TEST_FILENAME")
+
+    echo "file name $BATS_TEST_FILENAME"
+
+    source "$PROJECT_ROOT/k8s_helper.sh"
+
+    # set default image from the repo charts or take it as a parameters from CI
+    IMAGE_REPO=${IMAGE_REPO:-$(helm show values percona/pmm | grep 'repository:' | cut -d ':' -f 2 | xargs)}
+    IMAGE_TAG=${IMAGE_TAG:-$(helm show values percona/pmm | grep 'tag:' | cut -d ':' -f 2 | xargs)}
+}
+
+teardown() {
+
+    echo "cleanup"
+    helm list --short | xargs helm uninstall || true
+    kubectl delete pods,services,statefulsets,configmaps,secrets,serviceaccount --selector=app.kubernetes.io/name=pmm --force || true
+    delete_pvc || true
+    rm values.yaml || true
+}
+
+@test "add helm repo" {
+    helm repo add percona https://percona.github.io/percona-helm-charts/
+}
+
+@test "generate values.yaml" {
+    helm show values percona/pmm > values.yaml
+}
+
+@test "install/uninstall default chart and check connectivity" {
+    helm install pmm \
+        --set image.repository=$IMAGE_REPO \
+        --set image.tag=$IMAGE_TAG \
+        percona/pmm
+    wait_for_pmm
+
+    # depends on the driver, but probably local PVC wouldn't be cleaned up
+    # and pass would be set only during this first pvc init
+    # so always use new name if you want to provision PVC in helm install (pmmX)
+    local admin_pass=$(get_pmm_pswd)
+    local pmm_address=$(get_pmm_addr)
+
+    #encode pass, as it can have special characters
+    local encoded_u_p=$(echo -n admin:${admin_pass} | base64)
+
+    echo "curl -k -H 'Authorization: Basic ...' https://"${pmm_address}"/v1/version"
+    run bash -c "curl -sk -H 'Authorization: Basic ${encoded_u_p}' https://${pmm_address}/v1/version"
+    echo $output
+    [ "$status" -eq 0 ]
+    echo "${output}" | grep "full_version"
+
+    helm uninstall --wait --timeout 60s pmm
+    delete_pvc
+}
+
+@test "install/uninstall with parameter set in cli" {
+    helm install pmm1 \
+        --set image.repository=$IMAGE_REPO \
```

Comment:
how does it read the value from our pipelines?

Reply:
in GH actions there are inputs, and it got converted there and exported as env

Reply:
no, I meant, since you overwrote those variables in the setup, I believe we would not be able to use env. variables instead use again the same from the values file.

Reply:
I didn't ;) ` IMAGE_TAG=${IMAGE_TAG:-` minus here means only when not defined

Reply:
aah yes, I didn't read that carefully, got it